web-client/public/gene/info.css

```diff
@@ -40,16 +40,17 @@ sup {
   min-width: 100%;
 }
 
-
 #iframeMenu {
   position: absolute;
   min-height: 450px;
   min-width: 160px;
+  z-index: -1;
 }
 
 #iframeFooter {
   width: 100%;
 	display: block;
+  z-index: 0;
```

Comment:
I don't see how this fixes the issue. `z-index` is not used for layout changes. If this appears to fix things (and I don't think so, because I tried to apply the changes manually in the browser without a visible effect), then I think that appearance is coincidental.

The true way to fix this is to make the gene page layout structure match the GRNsight page layout structure. Upon inspection of the latter, the footer is actually part of the main section (`#mainContent`) on the GRNsight pages and so it retains a margin that offsets it from the sidebar. In the gene page, the footer is _both_ the sidebar and main content.

Based on this, it looks like the right first step is to put the footer iframe within `#mainPage`, which appears to be the closest equivalent, but in structure and layout, to `#mainContent` on the GRNsight pages. Once it is placed there, adjustments can then be made to ensure a better visual match.

@johnllopez616 remember that web page editing and layout is a big-picture activity. It can be counterproductive to make small pinpoint changes here and there when in reality, stepping back, there are broader structural issues that keep things from lining up (like what appears to be happening above). Remember also to not form some type of tunnel vision, where you might be so focused on a particular visual detail without realizing that changes are then happening to other parts of a page. The ultimate goal of web page design activities is to provide something that is simultaneously useful, informative, functional, and aesthetically pleasing. Maintaining all of these factors requires that you keep a birds-eye view of the page even while working on details, in order to avoid regressions and to catch smaller glitches that might not have been mentioned explicitly.

Reply:
@johnllopez616 made changes in gh-pages that interact with the z-index. He says he will document that.